module/blobs/blob_store.go

```diff
@@ -0,0 +1,82 @@
+package blobs
+
+import (
+	"context"
+	"errors"
+
+	"github.com/ipfs/go-cid"
+	"github.com/ipfs/go-datastore"
+	blockstore "github.com/ipfs/go-ipfs-blockstore"
+)
+
+type Blobstore interface {
```

Comment:
just curious:  Why do we copy [Blockstore](https://pkg.go.dev/github.com/ipfs/go-ipfs-blockstore#Blockstore) interface instead of using it directly?

Reply:
Block is confusing terminology in a blockchain codebase